### ISSUE TITLE
Getting an empty plan comment in the terraform pr.

### ISSUE THREAD

**Issue by tamccall** [robburger/terraform-pr-commenter#5]
![Screenshot from 2021-01-29 09-13-01](https://user-images.githubusercontent.com/8611053/106305981-40bd1900-6212-11eb-8ce6-51e7698ff885.png)
Using the example workflow file from: https://github.com/marketplace/actions/terraform-pr-commenter

Can see in the actions where my plan was run and there i can see the relevant updates

**Comment by Fresa** [comment]
Yeah something is wrong with the parsing of the input somewhere. If `TF_IN_AUTOMATION: true` is not set I get the following output:

![image](https://user-images.githubusercontent.com/1190300/109044042-39cedc80-76d2-11eb-8bd1-85416eca30e0.png)

I cherry picked the plan parsing parts from 
https://github.com/robburger/terraform-pr-commenter/blob/b7622fb18aa70b286ca8e95dbc90bd6ae197d731/entrypoint.sh#L33
and 
https://github.com/robburger/terraform-pr-commenter/blob/b7622fb18aa70b286ca8e95dbc90bd6ae197d731/entrypoint.sh#L186-L192
which became
```
INPUT=$(terraform plan | sed 's/\x1b\[[0-9;]*m//g')
CLEAN_PLAN=$(echo "$INPUT" | sed -n '/Refreshing state\.\.\./!p') # Strip refresh section
CLEAN_PLAN=$(echo "$CLEAN_PLAN" | sed -nr '/-{72}/q;p') # Ignore everything after the 72 dashes (happens when saving a plan to file)
CLEAN_PLAN=${CLEAN_PLAN::65300} # GitHub has a 65535-char comment limit - truncate plan, leaving space for comment wrapper
CLEAN_PLAN=$(echo "$CLEAN_PLAN" | sed -r 's/^([[:blank:]]*)([-+~])/\2\1/g') # Move any diff characters to start of line
CLEAN_PLAN=$(echo "$CLEAN_PLAN" | sed -r 's/^~/!/g') # Replace ~ with ! to colourise the diff in GitHub comments
echo "$CLEAN_PLAN"
```

and ran it locally, and the output looked correct.

The input to Docker looks fine as far as I can tell when viewing it in the output of the Post Plan step.

Maybe `steps.plan.outputs.stdout` formats weirdly or something happens with the formatting when sending it to Docker?

**Comment by robburger** [comment]
Apologies, these notifications went to spam and I missed this thread!

@tamccall or @Fresa would you be able to confirm which version of Terraform you're using in the preceding steps?

My immediate suspicion is that this is related to Terraform v0.14's change to the way the output is formatted after a plan (they dropped the 72 `-`'s.) I've not yet moved my personal Terraform config over from v0.13 to v0.14 yet and it seems to still be running fine on both v0.12 and v0.13 - at least in the quick checks that I've done now.

I'll have a look in the next day or two to see if there's a backwards-compatible way of stripping off that refresh section.

**Comment by Fresa** [comment]
v0.14.7

**Comment by tamccall** [comment]
```
      - name: Setup Terraform
        uses: hashicorp/setup-terraform@v1
        with:
          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
          terraform_version: 0.14.5
```

**Comment by robburger** [comment]
This should now be resolved in `v1.3.0`. The default `v1` tag has also been moved to the latest commit https://github.com/robburger/terraform-pr-commenter/commit/c4553968e1959190073eb056c9fc8a6582cb06b7

I've tested in a few plans on v0.12/v0.13/v0.14 and it seems to work consistently, but please feel free to re-open this issue if there's anything weird going on with the formatting or I've missed a test case!

**Comment by Fresa** [comment]
Can confirm that it works like a charm now! Great work @robburger! 🎉
